### PR TITLE
Mscclpp CI test

### PR DESCRIPTION
**Pull request by corey-derochie-amd**
# DRAFT for CI purposes only

### CODE REVIEW

**Comment by nileshnegi** [comment]
is MSCCLPP disabled by default?

File: CMakeLists.txt [hunk 1]
```diff
@@ -27,7 +27,6 @@ option(BUILD_TESTS                             "Build unit test programs"
 option(COLLTRACE                               "Collective Trace Option"                       ON)
 option(ENABLE_MSCCL_KERNEL                     "Enable MSCCL while compiling"                  ON)
 option(ENABLE_MSCCLPP                          "Enable MSCCL++"                                OFF)
```
Comment (corey-derochie-amd) [inline CMakeLists.txt]:
```suggestion
option(ENABLE_MSCCLPP                          "Enable MSCCL++"                                ON)
```

**Comment by corey-derochie-amd** [comment]
> is MSCCLPP disabled by default?

Yes, forgot to change that. Fixed in the PR.

**Comment by nileshnegi** [comment]
> > is MSCCLPP disabled by default?
> 
> Yes, forgot to change that. Fixed in the PR.

will need an edit in `install.sh` too

**Comment by corey-derochie-amd** [comment]
> > > is MSCCLPP disabled by default?
> > 
> > 
> > Yes, forgot to change that. Fixed in the PR.
> 
> will need an edit in `install.sh` too

`install.sh` was never modified, so it shouldn't require an edit.

**Comment by wenkaidu** [comment]
Let's ask CQE to run a staging cycle with this commit before merging.

**Comment by corey-derochie-amd** [comment]
> Let's ask CQE to run a staging cycle with this commit before merging.

This PR isn't meant to be merged, it's for CI testing. Although I guess we could merge it if we were happy with it.

**Pull request closed without merging**